packages/headless/src/features/commerce/context/cart/cart-state.ts

```diff
@@ -1,6 +1,10 @@
 import {CartItemParam} from '../../../../api/commerce/commerce-api-params';
 
 export interface CartItemWithMetadata extends CartItemParam {
+  /**
+   * The product unique identifier of the cart item.
+   */
```

Comment:
We use this in the `CartItem`. Both seem like good description, but we could be consistent.
```suggestion
  /**
   * The unique identifier of the product.
   */
```